src/feature-viewer.js

```diff
@@ -1025,8 +1065,27 @@ function createFeature(sequence, div, options) {
                 })
                 forcePropagation(histog);
             },
-            bar: function (object, position) {
+
+            bar: function (object, position,onClick) {
                 if (object.fill === undefined) object.fill = true;
+                if(typeof object.summaryView === 'undefined' || object.summaryView === null) object.summaryView = false;
+                if(typeof object.summaryViewProperties === 'undefined' || object.summaryViewProperties === null) object.summaryViewProperties = {};
+                if(typeof object.summaryViewProperties.buttonColor === 'undefined' || object.summaryViewProperties.buttonColor === null) object.summaryViewProperties.buttonColor = "#5789d4";
+                if(typeof object.summaryViewProperties.buttonTextColor === 'undefined' || object.summaryViewProperties.buttonTextColor === null) object.summaryViewProperties.buttonTextColor = "#ffffff";
+                if(typeof object.summaryViewProperties.buttonLabel === 'undefined' || object.summaryViewProperties.buttonLabel === null) object.summaryViewProperties.buttonLabel = "Click Here to Load All Data";
+                if(typeof object.summaryViewProperties.position === 'undefined' || object.summaryViewProperties.position === null) object.summaryViewProperties.position = "left";
+
+                let btnPosition = 0;
+                if(object.summaryViewProperties.position === 'left'){
+                    btnPosition = 2;
+                } else if(object.summaryViewProperties.position === 'right'){
+                    btnPosition = (fvLength/6) *  4.7 - 2;
+                } else if(object.summaryViewProperties.position === 'center'){
+                    btnPosition = (fvLength/6) * 2.5;
```

Comment:
Why the position calculation changes between graph types?

Reply:
This is due to height differences in two graphs. Bar graph has default hight as 5 and Rect graph has default hight of 1.